doc/users/next_whats_new/mathtext_supports_text.rst

```diff
@@ -1,12 +1,12 @@
 ``mathtext`` now supports ``\text``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``\text`` can be used to obtain upright text within an equation.
+``\text`` can be used to obtain upright text within an equation and to get a plain -.
 
 .. plot::
     :include-source: true
     :alt: Illustration of the newly added \text command, showing that it renders as normal text, including spaces, despite being part of an equation. Also show that a dash is not rendered as a minus when part of a \text command.
 
-import matplotlib.pyplot as plt
-plt.text(0.1, 0.5, r"$a = \sin(\phi) \text{ such that } \phi = \frac{x}{y}$")
-plt.text(0.1, 0.3, r"$\text{dashes (-) are retained}$")
+    import matplotlib.pyplot as plt
```

Comment:
You need to make this a code block

```suggestion
::

    import matplotlib.pyplot as plt
```

Reply:
I think that it will work as is. Compare to e.g. https://raw.githubusercontent.com/matplotlib/matplotlib/main/doc/users/next_whats_new/pie_hatch.rst which renders as https://matplotlib.org/devdocs/users/next_whats_new/pie_hatch.html

But building the documentation locally to confirm.

Reply:
(I realized that the hatch-parameter for pie is in the 3.7-release notes, so probably it should not hang around anymore? @story645 @QuLogic ? I can of course fix it, just want to see that I have not missed anything here...)